prov/efa/src/efa_prov_info.c

```diff
@@ -471,12 +471,18 @@ int efa_prov_info_alloc(struct fi_info **prov_info_ptr,
 
 	if (ep_type == FI_EP_RDM) {
 		prov_info->caps	= EFA_RDM_CAPS;
+
 		/* Claim RMA support in the efa-direct path only if read, write
-		 *  and unsolicited write are all available */
-		if (efa_device_support_rdma_read() &&
-			efa_device_support_rdma_write() &&
-			efa_device_support_unsolicited_write_recv())
+		 *  and unsolicited write are all available
+		 */
+#if HAVE_CAPS_UNSOLICITED_WRITE_RECV
```

Comment:
I'm not sure I understand why this change is necessary.  Could really use some documentation in the comment.

Reply:
`EFADV_DEVICE_ATTR_CAPS_UNSOLICITED_WRITE_RECV` is not defined in older versions of rdma-core and that causes compilation failures. Will add comment.